src/app-layout/defaults.ts

```diff
@@ -57,6 +56,5 @@ export function applyDefaults(
     maxContentWidth: stateFromProps.maxContentWidth ?? contentTypeDefaults.maxContentWidth,
     minContentWidth: stateFromProps.minContentWidth ?? contentTypeDefaults.minContentWidth,
     navigationOpen: stateFromProps.navigationOpen ?? contentTypeDefaults.navigationOpen,
-    toolsOpen: stateFromProps.toolsOpen ?? contentTypeDefaults.toolsOpen,
```

Comment:
None content types define default tools state. Let's remove this and simplify our code